lambda/script.py

```diff
@@ -0,0 +1,121 @@
+import json
+import os
+
+# from pprint import pprint
+from typing import Dict, List, Optional
+
+import boto3  # type: ignore
+import googleapiclient.discovery  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+
+def get_env_var(env) -> Optional[str]:
```

Comment:
Please add doc strings for all functions.

Reply:
Just done!